### ISSUE TITLE
Info bar

### ISSUE THREAD

**Issue by mdobes** [brunochanrio/MochiTIF#5]
The info bar on the top, where is info about channel is not disappearing when using arrow keys (up and down). To "remove" the bar I have to press return.
![image](https://github.com/brunochanrio/MochiTIF/assets/26965898/fcb06195-05b7-4eea-bf0a-f296ac0721d4)

**Comment by brunochanrio** [comment]
You probably have an accessibility service enabled

**Comment by mdobes** [comment]
Yes, I remapped "Live TV" button on the remote to open this app. Do you think you can help me with solving this issue? I use [this](https://play.google.com/store/apps/details?id=flar2.homebutton&hl=en_US) app for key mapping. Or should I contact the developers? Thanks.

**Comment by mdobes** [comment]
<img width="1137" alt="image" src="https://github.com/brunochanrio/MochiTIF/assets/26965898/e45b5e91-2757-46f0-8309-750d5a90bd0d">

Do you think you could provide a fix, please? I would appreciate that.

**Comment by jhedfors** [comment]
It would be great if we can turn off the Accessibility "features" that turn on when you have Accessibility turned on for ANY app.   I also want to use a button remapper app, but my issue is that it messes up the EPG scrolling.

@brunochanrio You may want to look a the code referenced here https://github.com/brunochanrio/MochiTIF/issues/1#issuecomment-1900456882, as it may be the culprit.

**Comment by ScriptNewbie** [comment]
> It would be great if we can turn off the Accessibility "features" that turn on when you have Accessibility turned on for ANY app. I also want to use a button remapper app, but my issue is that it messes up the EPG scrolling.
> 
> @brunochanrio You may want to look a the code referenced here [#1 (comment)](https://github.com/brunochanrio/MochiTIF/issues/1#issuecomment-1900456882), as it may be the culprit.

It's a shame it's not fixed, as it seems to be an easy fix. I would love to test my theory, unfortunately I have no experience what so ever with building Android apps from Java source, especially that "Live TV example app" seems to be a system app. I spend a little bit of time trying but realised it's not worth it.
